airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -21,6 +21,11 @@ class SlackStream(HttpStream, ABC):
     url_base = "https://slack.com/api/"
     primary_key = "id"
     page_size = 100
+    channel_filter = []
+
+    def __init__(self, channel_filter: List[str] = [], **kwargs):
+        self.channel_filter = channel_filter
```

Comment:
nit: I don't think this `channel_filter` is meant to be used in all the child streams (e.g `Users`), you could create an intermediate class specifically for channel-related streams.

Reply:
done